fix error "Cannot read property '4' of null in gplay.app"

icarcal opened the pull request:
This PR fixes the following issue:
#387 

I have updated the following methods:
`.app`

This change fix the error "Cannot read property '4' of null in gplay.app".
The structure of the arrays at gplay store have changed, so I just updated the MAPPINGS variable

I have added some debugging too.

icarcal commented:
The tests are failing due to other issues, unrelated to this one.
I'm going to send another PR to fix the tests.

facundoolano commented:
There's a style error which is from the new code. Also, can you update the app test so it breaks for the no comments case?

icarcal commented:
@facundoolano I've rebased my branch with the latest updates in the `dev` branch, and fixed the style error.
Now the errors are related to the store itself.
I'm going to take a better look and see if I can fix those tests in another PR.
